tox.ini

```diff
@@ -16,6 +18,7 @@ extras =
     test
 description = Run unit tests with pytest. This is a special environment that does not get a name, and
               can be referenced with "py".
+skipsdist = False
```

Comment:
Is this necessary, who put it there?

Reply:
I put it there. Was testing it..not required. Taking it off.